cmd/dumpling/main.go

```diff
@@ -127,6 +128,7 @@ func main() {
 	pflag.StringVar(&csvDelimiter, "csv-delimiter", "\"", "The delimiter for values in csv files, default '\"'")
 	pflag.StringVar(&outputFilenameFormat, "output-filename-template", "", "The output filename template (without file extension)")
 	pflag.BoolVar(&completeInsert, "complete-insert", false, "Use complete INSERT statements that include column names")
+	pflag.StringToStringVar(&sessionParams, "params", nil, `Extra session params used in dumping process, accept format: --params "character_set_client=latin1,character_set_connection=latin1"`)
```

Comment:
```suggestion
	pflag.StringToStringVar(&sessionParams, "params", nil, `Extra session variables used while dumping, accepted format: --params "character_set_client=latin1,character_set_connection=latin1"`)
```

Reply:
ok. Do we need to change the argument name `params` to `session-variables` or something else?

Reply:
i think it's fine.

Reply:
OK. Some other code has been changed since the last commit. PTAL again.